### PR TITLE
Report input positions distribution in EXPLAIN ANALYZE VERBOSE

### PR DESCRIPTION
**Pull request by sopel39**
Example output
```
 └─ TableScan[hive:tpch_sf10_orc:lineitem_part, grouped = false]
        Layout: []
        Estimates: {rows: 32639679 (0B), cpu: 0, memory: 0B, network: 0B}
        CPU: 3.62s (88.81%), Scheduled: 13.36m (99.94%), Output: 34156833 rows (0B)
        metrics:
          'Input distribution' = TDigestHistogram{count=1430.00, p01=3102.00, p05=14835.88, p10=24473.31, p25=24781.22, p50=24913.10, p75=25033.02, p90=25128.67, p95=25
        Input avg.: 23885.90 rows, Input std.dev.: 16.87%
        shipdate:date:PARTITION_KEY
            :: [[1995-01-02, 1998-12-01]]

 └─ InnerJoin[("nationkey" = "nationkey_0")][$hashvalue, $hashvalue_7]
    │   Layout: []
    │   Reorder joins cost : {rows: 100000 (0B), cpu: 1.72M, memory: 225B, network: 225B}
    │   Estimates: {rows: 100000 (0B), cpu: 5.15M, memory: 450B, network: 450B}
    │   CPU: 97.00ms (61.78%), Scheduled: 346.00ms (51.87%), Output: 100000 rows (0B)
    │   Left (probe) metrics:
    │     'Input distribution' = TDigestHistogram{count=1.00, p01=100000.00, p05=100000.00, p10=100000.00, p25=100000.00, p50=100000.00, p75=100000.00, p90=100000.00, p
    │   Right (build) metrics:
    │     'Input distribution' = TDigestHistogram{count=16.00, p01=0.00, p05=0.00, p10=0.00, p25=0.00, p50=1.00, p75=4.00, p90=4.00, p95=4.00, p99=4.00, min=0.00, max=4
    │   Left (probe) Input avg.: 100000.00 rows, Input std.dev.: 0.00%
    │   Right (build) Input avg.: 1.56 rows, Input std.dev.: 98.55%
    │   Collisions avg.: 0.80 (155.86% est.), Collisions std.dev.: 111.80%
```
cc @martint

### CODE REVIEW

**Comment by sopel39** [comment]
@martint now standard metrics I used. I skipped `pruneIntermediateXX` methods. We might return to the idea if we find query JSON size problematic

File: core/trino-main/src/main/java/io/trino/sql/planner/planprinter/HashCollisionPlanNodeStats.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public HashCollisionPlanNodeStats(
             long planNodeOutputPositions,
             DataSize planNodeOutputDataSize,
             DataSize planNodeSpilledDataSize,
-            Map<String, OperatorInputStats> operatorInputStats,
+            Map<String, OperatorInputStats> operatorStats,
```
Comment (losipiuk) [inline core/trino-main/src/main/java/io/trino/sql/planner/planprinter/HashCollisionPlanNodeStats.java]: Should class be renamed too?

Comment (losipiuk) [reply core/trino-main/src/main/java/io/trino/sql/planner/planprinter/HashCollisionPlanNodeStats.java]: Oh I see you rename it in other commit. Should be here.

Comment (sopel39) [reply core/trino-main/src/main/java/io/trino/sql/planner/planprinter/HashCollisionPlanNodeStats.java]: will squash them

**Review by losipiuk — COMMENTED** [review]
LGTM

File: lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/metrics/TestMetrics.java [hunk 2]
```diff
@@ -61,7 +61,8 @@ public void testMergeHistogram()
         assertThat(merged.getTotal()).isEqualTo(3L);
         assertThat(merged.getPercentile(0)).isEqualTo(5.0);
         assertThat(merged.getPercentile(100)).isEqualTo(10.0);
-        assertThat(merged.toString()).matches("TDigestHistogram\\{count=3.0, p0=5\\.0, p10=5\\.0, .*, p90=10\\.0, p100=10\\.0\\}");
+        assertThat(merged.toString())
+                .matches("TDigestHistogram\\{count=3\\.00, p01=5\\.00, p05=5\\.00, p10=5\\.00, p25=5\\.00, p50=7\\.50, p75=10\\.00, p90=10\\.00, p95=10\\.00, p99=10\\.00, min=5\\.00, max=10\\.00\\}");
```
Comment (martint) [inline lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/metrics/TestMetrics.java:65]: The `TDigestHistogram` part in the string is unnecessary.

**Review by martint — APPROVED** [review]

**Pull request merged**